provisioning/miniprov/hare_mp/utils.py

```diff
@@ -166,16 +166,16 @@ def is_motr_component(self, machine_id: str) -> bool:
         return rc
 
     @func_log(func_enter, func_leave)
-    def is_s3_component(self, machine_id: str) -> bool:
+    def is_client_component(self, machine_id: str, client: str) -> bool:
```

Comment:
By having this generic function, can we now remove these two functions:
is_motr_component
is_s3_component